### PR TITLE
Sophos UTM - add module to handle utm aaa group entries

### PR DESCRIPTION
**Pull request by MatrixCrawler**
##### SUMMARY
This PR will add a module to the sophos utm namespace to handle the aaa group entries.

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
- utm_aaa_group

### CODE REVIEW

File: lib/ansible/modules/web_infrastructure/sophos_utm/utm_aaa_group.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+description:
+    - Create, update or destroy a reverse_proxy frontend entry in SOPHOS UTM.
+    - This module needs to have the REST Ability of the UTM to be activated.
+
+version_added: "2.8"
+
+options:
+    name:
+        description:
+          - The name of the object. Will be used to identify the entry
```
Comment (Akasurde) [inline lib/ansible/modules/web_infrastructure/sophos_utm/utm_aaa_group.py]:
```suggestion
          - The name of the object. Will be used to identify the entry.
```

**Review by Akasurde — COMMENTED** [review]
Rest LGTM.

**Comment by maunzCache** [comment]
@MatrixCrawler Please rebase the branch or close this PR to support my duplicate so we can shipit ;)

**Comment by gundalow** [comment]
https://github.com/ansible/ansible/pull/49860 has been merged and will be released in Ansible 2.8.

**Pull request closed without merging**